relay-dynamic-config/src/defaults.rs

```diff
@@ -338,6 +338,23 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                     .always(),
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/count_per_segment@none".into(),
```

Comment:
Did you add this string to https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/strings.py? Please make sure that's deployed before merging this PR.

Reply:
gosh darn it, here you go! https://github.com/getsentry/sentry/pull/62079